### ISSUE TITLE
Make commands more user friendly

### ISSUE THREAD

**Issue by J-T-Nelson** [erincayaz/obsidian-colored-text#19]
Not sure if this is a problem specific to me, or if anyone else has gotten the issue. 

As described in the title and as you can see in the attached image, all of the color blocks are black in the bottom right of the screen. I have restarted Obsidian multiple times to no avail. 

For system details, 
I am running v1.3.5 of Obsidian on Windows 10. 

If you would like any other information, let me know. I have not tried restarting my computer yet, as I am amidst some things I don't want to further interrupt. Will update if I see the plugin work in the future.

![image](https://github.com/erincayaz/obsidian-colored-text/assets/38541965/7d7c99ef-f7fe-40ba-9862-24029b98a3bb)

**Comment by juestchaos** [comment]
me too

**Comment by DominicDevelop** [comment]
Hey yout two,
you have to use the 4th hotkey you can set here
![grafik](https://github.com/erincayaz/obsidian-colored-text/assets/89411913/7000291e-3c69-448d-a456-3d2912f62175)

By using that hotkey you can select colors :) Hope that helps.

Lovely Greetings
Dominic

**Comment by erincayaz** [comment]
Can you let me know if using hotkey works for you?

**Comment by J-T-Nelson** [comment]
> Can you let me know if using hotkey works for you? 

The hotkey does work for me. I wasn't clear on what the hotkey was intended to do as "Get Color Input" isn't very revealing. I figured that meant I could select a color using my cursor like a typical color picker tool. Beyond that, starting with all black, and no way to alter the colors from the settings menu gave me the impression of a bug, or an incomplete system, which is why I decided to not continue exploring features.  

I would suggest changing the command to be more directly descriptive, or the documentation to offer a simple example (like a gif showing palette modification, (I recognize there is a gif, though its not clear what command you're using) ), or the plugin settings menu to allow for selection of ones color palette within the settings menu itself. Any of these changes would enhance the usability of the plugin from the start, and would ease a users experience of picking up and making use of the plugin! 

As for renaming the command, I would suggest, "Select Color Palette", "Alter Color Palette", or something less ambiguous than the current command name. Further I would suggest changing the name of the "Add the colored text" to something clear and simple like "Color Text", "Change Color", or "Color Highlighted Text". 

I appreciate the plugin, and hope you understand my feedback isn't mean spirited, but intended to help the plugin be more easily usable and understandable for new users!
